app/views/competitions/_nav.html.erb

```diff
@@ -50,11 +50,16 @@
               ] : [])
           }
           if @competition.use_wca_registration?
+            pending_registrations = if @competition.uses_new_registration_service?
+                                      Microservices::Registrations.registrations_by_competition(@competition.id, 'pending', cache: false)
```

Comment:
This is lacking a `.length` or something. Also, why are you not caching?

Reply:
if we cache, this will not be accurate right?

Reply:
Oh, the name `cache: true` might be a bit misleading here. It controls whether the data returned from the Microservice should also be _written to_ the caching tables or not.